scripts/utils.sh

```diff
@@ -47,7 +47,7 @@ kubectl_retry() {
 	# log stderr and empty the tmpfile
 	cat "${stderr}" > /dev/stderr
 	true > "${stderr}"
-	echo "kubectl_retry ${*} failed, will retry in ${KUBECTL_RETRY_DELAY} seconds"
+	echo "$(date): 'kubectl_retry ${*}' failed (${retries}/${KUBECTL_RETRY}), will retry in ${KUBECTL_RETRY_DELAY} seconds" > /dev/stderr
```

Comment:
this log was written to the temporary `${stderr}` file, now it'll be written to stderr so that the Jenkins log catches it